src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,17 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
+        //assertTrue(false);
         // add your tests here
+        int[][] test = {{},{1},{2,1},{3,1,2},{4,2,3,1},{5,1,2,4,3}};
+        int[][] target = {{},{1},{1,2},{1,2,3},{1,2,3,4},{1,2,3,4,5}};
+        for (int i = 0; i < test.length; i++) {
+            Problem2.bubbleSort(test[i]);
+            assertArrayEquals(test[i],target[i]);
```

Comment:
the order of argument is reversed. 1st argument is the expected value, and 2nd arg is the actual value.